frontend/public/locales/en/apply.json

```diff
@@ -68,5 +68,37 @@
       "empty-radio": "Please select a value",
       "empty-field": "Please enter a value"
     }
+  },
+  "confirm": {
+    "page-title": "Confirmation",
+    "p-one": "You can use the CDCP Status Checker or call 1-833-537-4342 to verify the status of your application. Your status should be available within the next 48 hours.",
+    "p-two": "If you haven't already, sign up for a <MSCALink>My Service Canada Account (MSCA)</MSCALink> — a secure online portal designed to streamline your application management process.",
+    "p-three": "Once your application is processed, you will be able to sign in to MSCA to view/update your personal information and view any letters sent to you by the Government of Canada regarding the CDCP.",
+    "p-four": "If you are deemed eligible, Sun Life will send you a welcome package by mail. Your welcome package will include your member card and details about the CDCP, including your coverage start date. The CDCP will not reimburse you for dental services received before that date.",
+    "p-five": "If you are deemed ineligible and circumstances change that may impact your eligibility for the CDCP, you can submit a new application.",
+    "p-six": "For more information, please contact <dentalContactUsLink>Service Canada.</dentalContactUsLink>",
```

Comment:
**Nitpick:** Avoid using the current order of text as i18n key names but rather a key name with semantic meaning instead. If in the future we have to re-arrange the order of the text, developers may not change the key name, and we could have something like `p-six` appearing before `p-four`.